content/PreparedStatement/PreparedStatement.facts.md

```diff
@@ -0,0 +1,62 @@
+### Facts for Prepared Statement lesson:
+
+A `prepared statement` is used to execute an SQL statement multiple times with the same or different parameter values. The execution of a prepared statement consists of using `prepare()` and `execute()` methods. 
+
+- The `PDO::prepare()` method prepares an SQL statement for execution using the `PDOStatement::execute()` method and returns a `PDOStatement` object.
+
+- The `PDOStatement::execute()` method executes a prepared statement and returns `true` on success or `false` on failure.
+
+- The `PDOStatement::fetchAll()` method fetches a result set and returns an array of the remaining rows in a result set.
```

Comment:
This is so confusing IMO. I mean, I'm not sure about `and returns an array of the remaining rows in a result set`. Could we simplify this one? Something like it fetches all result set rows.

Reply:
Also, did we inform learners what result set is already? Was this covered by `postgresql-introduction` course?

Reply:
We need the to include `fetch` and `fetchColumn`

Reply:
For the in depth discussion of `fetch` and `fetchColumn` will be discussed in the later lessons. I discussed it with ms. ella the purpose of the fetchAll() in this lesson. We agreed to generally just define the usage of `fetchAll()` so that we can display result set rows to learners for this lesson. Let me know what you think manager. Thanks.

![TableOfContentDbInPhp](https://user-images.githubusercontent.com/38550259/64308689-b779c680-cfcc-11e9-83d0-1fc79cd30595.jpg)

And for the requirements for Database in PHP are the following which includes postgresql-intermediate: 
![DBinPHPCourseIntroduction](https://user-images.githubusercontent.com/38550259/64308701-c2345b80-cfcc-11e9-8f07-9258c5e85b21.jpg)